src/main/java/org/folio/services/budget/BudgetService.java

```diff
@@ -60,7 +60,8 @@ private SharedBudget mergeBudgets(SharedBudget sharedBudget, Budget budgetFromSt
       .withEncumbered(budgetFromStorage.getEncumbered())
       .withOverEncumbrance(budgetFromStorage.getOverEncumbrance())
       .withOverExpended(budgetFromStorage.getOverExpended())
-      .withNetTransfers(budgetFromStorage.getNetTransfers());
+      .withNetTransfers(budgetFromStorage.getNetTransfers())
+      .withTotalFunding(budgetFromStorage.getTotalFunding());
```

Comment:
Looks like we need to multiply : budget.allocated + budget.netTransfers directly, because totalFunding is not stored in the DB